cupy/fallback_mode/fallback.py

```diff
@@ -0,0 +1,107 @@
+"""
+Main fallback class.
+"""
+import types
+import cupy as cp
+import numpy as np
+
+
+from cupy.fallback_mode.utils import _call_cupy
+from cupy.fallback_mode.utils import _call_numpy
+
+
+class _RecursiveAttr:
+    """
+    RecursiveAttr class to catch all attributes corresponding to numpy,
+    when user calls fallback_mode. numpy is an instance of this class.
+    """
+
+    def __init__(self, numpy_object, cupy_object):
+
+        self._numpy_object = numpy_object
+        self._cupy_object = cupy_object
+
+    @property
+    def _cupy_module(self):
```

Comment:
In the current implementation, this module seems not needed because it is just an alias of `self._cupy_object`.

Reply:
Okay removed it